v63004/u_inref/gtm8916.csh

```diff
@@ -0,0 +1,61 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+
```

Comment:
Need to do the following. Or else we could have a frozen instance (due to fake enospc) triggered by an update process which will never be unfrozen because it was kill -9ed by the test.

unsetenv gtm_test_freeze_on_error